### ISSUE TITLE
Inaccurate HVAC heating consumption evolution for building rotation

### ISSUE THREAD

**Issue by msfer86** [NREL/EnergyPlus#9834]
Issue overview
--------------
Assessing the annual HVAC heating and (total) cooling consumption (using the ideal loads air system object) for different building orientations, covering the 360º-span, in a simple squared-shaped building, results in unexpected spikes in the heating consumption evolution, while the cooling consumption curve is smooth as anticipated (see Fig. 1, where the black curve corresponds to the total HVAC energy consumption: heating + cooling).
![Fig  1](https://user-images.githubusercontent.com/45457317/217277893-8d93ae3b-d677-4ab1-b53f-ef0eb342dc91.png)

Even more odd is the fact that no symmetry is observed in the heating consumption curve when rotating +- 90° from any given angle. There seems to exist some random effect on the heating energy consumption.

This occurs while using the weather file FRA_IF_Paris-Le.Bourget.AP.071500_TMYx.2004-2018.epw attached, but also happens with others. The building (Building.idf attached) consists of a parallelepiped single zone. It does not contain any windows, which may simplify the analysis of this issue.

Moreover, if the sun and wind effects are disabled in the simulation, the heating and cooling consumption evolution across the 360º-span should result in horizontal lines, since only the temperature effect is considered. However, it is not the case for the heating consumption (see Fig. 2). 
![Fig  2](https://user-images.githubusercontent.com/45457317/217277956-5404d013-1616-4c69-be9f-b7b41be32f9a.png)

Attachments:
[Archive.zip](https://github.com/NREL/EnergyPlus/files/10676124/Archive.zip)

### Details
Some additional details for this issue (if relevant):
 - Windows 11 64-bit (10.0.22621) & Mac OS 13.1 (22C65)
 - EnergyPlus v22.2

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

@eugeniorodrigues

**Comment by Nigusse** [comment]
Just by looking at the plots the swing between peak and bottom annual heating energy consumption difference (42.27 - 42.41) is in the order of 0.1%. I would not characterize such a small per-cent difference as inaccurate. Although, I am curious as what explains the annual heating energy consumption slight sensitivity to orientation relative to that of the cooling.

**Comment by msfer86** [comment]
@Nigusse, I agree on the small percent difference; however, this did not happen in previous E+ versions. And, oddly enough, only occurs for heating.
